classes/Customer.php

```diff
@@ -610,6 +610,9 @@ public static function customerHasAddress($idCustomer, $idAddress)
         return self::$_customerHasAddress[$key];
     }
 
+    /**
+     * Reset static cache
```

Comment:
Is this comment really useful as the function name is `resetStaticCache` ? 😄

Reply:
comment removed @matks